scripts/gha/build_testapps.py

```diff
@@ -558,24 +571,80 @@ def _build_ios(
     _run(
         xcodebuild.get_args_for_build(
             path=xcode_path,
-            scheme=api_config.scheme,
+            scheme=api_config.ios_scheme,
             output_dir=build_dir,
-            ios_sdk=_IOS_SDK_SIMULATOR,
+            apple_platfrom=_IOS,
+            apple_sdk=_IOS_SDK_SIMULATOR,
             configuration="Debug"))
 
   if _IOS_SDK_DEVICE in ios_sdk:
     _run(
         xcodebuild.get_args_for_build(
             path=xcode_path,
-            scheme=api_config.scheme,
+            scheme=api_config.ios_scheme,
             output_dir=build_dir,
-            ios_sdk=_IOS_SDK_DEVICE,
+            apple_platfrom=_IOS,
+            apple_sdk=_IOS_SDK_DEVICE,
             configuration="Debug"))
 
     xcodebuild.generate_unsigned_ipa(
         output_dir=build_dir, configuration="Debug")
 
 
+def _build_tvos(
+    sdk_dir, xcframework_exist, project_dir, repo_dir, api_config, tvos_sdk):
+  """Builds an iOS application (.app, .ipa or both)."""
+  if not xcframework_exist:
+    _build_xcframework_from_repo(repo_dir, "tvos", "tvos_build", api_config)
```

Comment:
Should we use a variable for the build directory `tvos_build` as it is being hardcoded in many places?

Reply:
Good point!